content/blog/2023/a-nextflow-docker-murder-mystery-the-mysterious-case-of-the-oom-killer.md

```diff
@@ -0,0 +1,152 @@
+title=A Nextflow-Docker Murder Mystery: The mysterious case of the “OOM killer”
+date=2023-06-19
+type=post
+description=Helpful tips for troubleshooting out-of-memory errors in containerized execution environments.
+image=img/a-nextflow-docker-murder-mystery-the-mysterious-case-of-the-oom-killer.jpg
+tags=nextflow
+status=published
+author=Graham Wright
+icon=graham.jpg
+~~~~~~
+## Troubleshooting out-of-memory errors with Nextflow and Docker
+
+*By: Graham Wright, Solutions Architect at Seqera Labs*
```

Comment:
I don't think we usually have this line? I think the front matter `author` and `icon` should be sufficient.